dot/network/transaction_test.go

```diff
@@ -31,29 +32,32 @@ func TestDecodeTransactionHandshake(t *testing.T) {
 func TestHandleTransactionMessage(t *testing.T) {
 	t.Parallel()
 
+	expectedMsgArg := &TransactionMessage{
+		Extrinsics: []types.Extrinsic{{1, 1}, {2, 2}},
+	}
+
+	ctrl := gomock.NewController(t)
+	transactionHandler := NewMockTransactionHandler(ctrl)
+	transactionHandler.EXPECT().
+		HandleTransactionMessage(peer.ID(""), expectedMsgArg).
+		Return(true, nil).MaxTimes(1)
```

Comment:
No need by default it's once. You can remove all `.MaxTimes(1)`

```suggestion
		Return(true, nil)
```